plugins/devtools-common/package.json

```diff
@@ -8,7 +8,8 @@
     "pluginPackages": [
       "@backstage/plugin-devtools",
       "@backstage/plugin-devtools-backend",
-      "@backstage/plugin-devtools-common"
+      "@backstage/plugin-devtools-common",
+      "@backstage/plugin-devtools-react"
```

Comment:
Why do we need this change? There's no other updates to the common package? Also, common packages are intended to work in both the frontend and backend adding a react package here would break that.

Reply:
This are automatic changes by [`yarn fix`](https://github.com/backstage/backstage/blob/4f83da1f7d2b2924e5c6a491c673291d9af8121a/package.json#L39) especially the `--publish` flag triggers this. 

My understanding is that this is used in internal tooling like `@backstage/cli` or the [Metadata Service](https://backstage.io/docs/backend-system/core-services/plugin-metadata/) 

See for example `catalog-common`

https://github.com/backstage/backstage/blob/4f83da1f7d2b2924e5c6a491c673291d9af8121a/plugins/catalog-common/package.json#L8-L14